Tests/TuistKitIntegrationTests/Utils/ProjectDescriptionHelpersBuilderTests.swift

```diff
@@ -25,17 +24,21 @@ final class ProjectDescriptionHelpersBuilderIntegrationTests: TuistTestCase {
     func test_build_when_the_helpers_is_a_dylib() throws {
         // Given
         let path = try temporaryPath()
+        subject = ProjectDescriptionHelpersBuilder(cacheDirectory: path)
         let helpersPath = path.appending(RelativePath("Tuist/ProjectDescriptionHelpers"))
         try FileHandler.shared.createFolder(helpersPath)
         try FileHandler.shared.write("import Foundation; class Test {}", path: helpersPath.appending(component: "Helper.swift"), atomically: true)
         let projectDescriptionPath = try resourceLocator.projectDescription()
 
         // When
+        _ = try subject.build(at: path, projectDescriptionPath: projectDescriptionPath)
```

Comment:
Is the purpose of adding two calls here to test out caching?

If so, perhaps we could do that in a separate test that tests for it a bit more explicitly?

e.g.

```swift

// When
let paths = try (0..<3).map { _ in try subject.build(at: path, projectDescriptionPath: projectDescriptionPath) }

// Then
// check all the paths are equal 
```

Reply:
That's a good idea. I changed the current test to follow your approach instead.